### PR TITLE
Add installation guide to README

### PR DESCRIPTION
**Pull request by yuriypavlyshak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+   For more details see [Requesting Authorization for Media Capture on iOS](https://developer.apple.com/documentation/avfoundation/capture_setup/requesting_authorization_for_media_capture_on_ios) (note that you don't need to use `AVCaptureDevice` API in your code, this is done by Whereby SDK).
+
+2. Add the audio and voice over IP background execution modes. In Xcode select your project file and then your app's target. In the target settings select the Signing & Capabilities tab. Add the Background Modes capability by clicking the "+" button in the top left corner. After adding the capability select the following values in the Background Modes section using the corresponding checkboxes:
+   - Audio, AirPlay, and Picture in Picture
+   - Voice over IP
+
+   For more details see [Configuring background execution modes](https://developer.apple.com/documentation/xcode/configuring-background-execution-modes).
+
+3. Disable Bitcode for your app's target. In Xcode select your project file and then your app's target. In the target settings select the Build Settings tab. Search for **Enable Bitcode** setting (`ENABLE_BITCODE`) and set its value to **No**.
```
Comment (remi-quinto) [inline README.md:70]: Is there any documentation we could link here, or explain why this is required maybe?

**Review by remi-quinto — APPROVED** [review]
I created a brand new app, followed the steps to get the SDK through cocoapods, and got it running pretty easily. 

One thing that is not included here is how to create the pod file (with `pod init`), in case you start with a brand new project and the fact that you have to open then `.xcworkspace` file. But we can consider that is something any iOS dev should know.

**Comment by yuriypavlyshak** [comment]
> One thing that is not included here is how to create the pod file (with `pod init`), in case you start with a brand new project and the fact that you have to open then `.xcworkspace` file. But we can consider that is something any iOS dev should know.

Yes, for the time being we can assume a developer is familiar with cocoapods setup or has set it up already in their project. Otherwise they will have to look it up (we link to cocoapods website that provides this info in the Prerequisites).

File: README.md (hunk 1 above)
Comment (yuriypavlyshak) [reply README.md:70]: I'll try to provide some more context in the new update.

**Pull request merged**